contrib/grafana-watcher/grafana/grafana.go

```diff
@@ -43,3 +44,15 @@ func (c *Clientset) Dashboards() DashboardsInterface {
 func (c *Clientset) Datasources() DatasourcesInterface {
 	return NewDatasourcesClient(c.BaseUrl, c.HTTPClient)
 }
+
+func doRequest(c *http.Client, req *http.Request) error {
+	resp, err := c.Do(req)
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return fmt.Errorf("doRequest failed, Code:%d, Status:%s", resp.StatusCode, resp.Status)
```

Comment:
Is `http.StatusOK` really the only valid status code the grafana api returns?

The error should probably rather be something like: "Unexpected status code returned from Grafana API (got: %d, expected: %d)".